samples/dashboards/data/googlesheets-tutorial/demo.js

```diff
@@ -72,6 +72,9 @@ const board = Dashboards.board('container', {
         },
         title: sheetTitle().original,
         dataGridOptions: {
+            credits: {
+                enabled: false
+            },
             editable: false
```

Comment:
You can remove this line because in the DataGrid2, cell editing is disabled by default, and the option is in a different place.

```suggestion
            }
```